cli/tests/test_windows_release_certification.py

```
@@ -41,10 +46,95 @@ def _step(job: dict[str, object], name: str) -> dict[str, object]:
+    struct.pack_into("<II", payload, directories + 32, certificate_offset, 16)
+    executable.write_bytes(payload)
+
+    helper = ROOT / "scripts" / "windows-authenticode.ps1"
+    command = (
+        ". $env:AUTHENTICODE_FIXTURE_HELPER; "
+        "Get-DefenseClawEmbeddedAuthenticodeCms $env:AUTHENTICODE_MALFORMED_PE"
+    )
+    env = os.environ.copy()
+    env["AUTHENTICODE_FIXTURE_HELPER"] = str(helper)
+    env["AUTHENTICODE_MALFORMED_PE"] = str(executable)
+    result = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)